package.json

```diff
@@ -1,4 +1,10 @@
 {
+  "scripts": {
+    "pretest": "rm -rf .jest-cache",
```

Comment:
If we need to clear the cache then we should be using `jest --clearCache`. It doesn't run the tests. It merely clears out the cache directory which is defined in the config.

Reply:
removed this change from this pr